js/initialize.jsx

```diff
@@ -0,0 +1,147 @@
+'use strict'
+
+var React = require('react/addons')
+var ipc = window.require('remote').require('ipc')
+
+var $ = require('jquery-bf')
+var bs = require('bootstrap')
+
+var Initform = React.createClass({
+  displayName: 'Initform',
+  mixins: [React.addons.LinkedStateMixin],
+
+  getInitialState: function () {
+    return {
+      error: null,
+      directory: '',
+      initState: 'uninitialized',
+      keysize: 2048
+    }
+  },
+
+  componentDidMount: function () {
+    var t = this
+    ipc.on('default-directory', function (dir) {
+      t.setState({directory: dir})
+    })
+
+    ipc.on('initializing', function () {
+      t.setState({initState: 'initializing'})
+    })
+
+    ipc.on('initialization-error', function (err) {
+      t.setState({
+        error: err,
+        initState: 'uninitialized'
+      })
+    })
+
+    ipc.on('initialization-complete', function () {
+      t.setState({initState: 'initialized'})
+    })
+  },
+
+  setKeySize: function (size) {
+    var t = this;
```

Comment:
Possiblly, but I think it may cause us to run into problems. Sometimes things have to be ignored. I think it might be better to just handle them in PRs. Though if there is a way to do warnings or something that would be neat.

The issue I see (and might be why this slipped in) is on line 105 of this file there is an HTML entitie. This may of failed standard and required a no veriffy to commit it.

There may also be a way around it with some variable trick.